### PR TITLE
feat: support iPadOS Split View and Slide Over

### PR DESCRIPTION
**Pull request by johnxnguyen**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Related Tickets & Documents

I saw this community post and thought I'd try to implement it. I simply changed the plist entry to not require fullscreen, and then adjusted the editor page so the collection view layout gets invalidated when the view controller's size changes. I did some tests changing different window sizes on different views within the app and didn't find any issues.

If there's anything that I'm missing, please let me know. Thanks!

https://community.anytype.io/t/split-view-on-ipad/23463

### Mobile & Desktop Screenshots/Recordings

Before:

https://github.com/user-attachments/assets/11fa9db8-a215-41c2-95b5-548d73e69894

After:

https://github.com/user-attachments/assets/26f460c4-8bb0-40d0-84ca-528aa819c7c6

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

File: Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageController.swift [hunk 1]
```diff
@@ -187,6 +187,16 @@ final class EditorPageController: UIViewController {
         }
     }
 
+    override func viewWillTransition(to size: CGSize, with coordinator: any UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+
+        // On iPadOS, when the app window changes from between fullscreen, split view, or slide over,
```
Comment (johnxnguyen) [inline Anytype/Sources/PresentationLayer/TextEditor/EditorPage/EditorPageController.swift:193]:
```suggestion
        // On iPadOS, when the app window changes between fullscreen, split view, or slide over,
```

**Comment by mgolovko** [comment]
Your changes have been passed to qa

**Comment by mgolovko** [comment]
Distribute error
Screenshot 2024-10-31 at 10 45 21

**Comment by johnxnguyen** [comment]
Hi @mgolovko @ignatovv , I've tried a fix that I found [here](https://stackoverflow.com/questions/74794155/invalid-bundle-because-your-app-supports-multitasking-on-ipad-you-need-to-incl), please let me know if this resolves that validation error.

**Pull request merged**